lib/models/user.js

```diff
@@ -333,50 +345,6 @@ function validateContainer (container) {
 
   return err;
 }
-/**
- * get backend url for a master pod instance url
- * @param  {String}   url        master pod instance url
- * @param  {String}   referer    http request referer (possible pod instance url) (can be undefined)
- * @param  {Function} cb  (err, host)
- *                        host format: <protocol>://<host>:<port>
- *                        will always exist
- */
-User.prototype.fetchBackendForUrl = function (url, referer, cb) {
```

Comment:
not used anymore (was used by navi)
https://github.com/search?utf8=%E2%9C%93&q=org%3ACodeNow+fetchBackendForUrl&type=Code&ref=searchresults

https://github.com/search?utf8=%E2%9C%93&q=org%3ARunnable+fetchBackendForUrl&type=Code&ref=searchresults